pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
 		setup.ensureSecretIsDeployed(t, credsName)
 	})
 }
+func TestOryIntegrationEvaluation(t *testing.T) {
+	skipTestIfDisabled(t)
 
-func (s *oryTest) getHorizontalPodAutoscaler(t *testing.T, name string) *autoscalingv1.HorizontalPodAutoscaler {
-	hpa, err := s.kubeClient.AutoscalingV1().HorizontalPodAutoscalers(namespace).Get(s.context, name, metav1.GetOptions{})
-	require.NoError(t, err)
+	if !isEvaluationProfile() {
+		t.Skipf("Integration tests disabled: skipping parts of test case '%s'", t.Name())
+	}
+
+	setup := newOryTest(t)
+	defer setup.contextCancel()
+
+	t.Run("ensure that ory-hydra hpa is not deployed", func(t *testing.T) {
+		name := "ory-hydra"
+		_, err := setup.getHorizontalPodAutoscaler(name)
+		require.Error(t, err)
+		setup.logger.Infof("HorizontalPodAutoscaler is not deployed")
+	})
+
+	t.Run("ensure that ory-oathkeeper hpa is not deployed", func(t *testing.T) {
+		name := "ory-oathkeeper"
+		_, err := setup.getHorizontalPodAutoscaler(name)
+		require.Error(t, err)
+		setup.logger.Infof("HorizontalPodAutoscaler is not deployed")
+	})
+
+	t.Run("ensure that ory-oathkeeper pod is deployed", func(t *testing.T) {
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=oathkeeper",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: oathkeeper", podsList.Items[0].Name)
+	})
+	t.Run("ensure that ory-hydra pod is deployed", func(t *testing.T) {
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=hydra",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: hydra", podsList.Items[0].Name)
+	})
+	t.Run("ensure that ory-hydra-maester pod is deployed", func(t *testing.T) {
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=hydra-maester",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: hydra-maester", podsList.Items[0].Name)
+	})
+	/*
+		t.Run("ensure that ory secrets are deployed", func(t *testing.T) {
```

Comment:
We can move it to the general Ory test, WDYT?

Reply:
moved